src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueClosureTests.cs

```diff
@@ -121,6 +121,59 @@ void F()
         }
 
         [Fact]
+        public void MethodWithStaticLocalFunction1()
+        {
+            var source0 = MarkedSource(@"
+using System;
+
+class C
+{
+    void F()
+    {
+        int x() <N:0>=> 1</N:0>;
```

Comment:
><N:0>=> 1</N:0> [](start = 16, length = 15)

This is odd. I'd expect the compiler to need mapping of the entire LocalFunctionStatement, not the arrow expression. In the IDE we match the statement, not the arrow expression, right? So this needs to match.

Reply:
Thank you! Incorrect match seems to be the root cause of  unexpected tests results.

---
In reply to: [135909017](https://github.com/dotnet/roslyn/pull/21777#discussion_r135909017) [](ancestors = 135909017)